ui/app/serializers/allocation.js

```diff
@@ -35,6 +35,9 @@ export default ApplicationSerializer.extend({
     // TEMPORARY: https://github.com/emberjs/data/issues/5209
     hash.OriginalJobId = hash.JobID;
 
+    hash.ModifyTimeNanos = hash.ModifyTime % 1000000;
```

Comment:
Ever think of using exponential notation for this or a constant of some sort? Could just be me, but counting the zeros makes me squint.

Reply:
Yeah, I suppose it's about time to turn this into a function. Something like

```js
const { datetime, nanos } = nsToDate(hash.ModifyTime);
```